src/common/components/header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import {Image, Flex, Button, Heading, HStack} from "@chakra-ui/react";
+
+function Header() {
+    return (
+        <HStack bg="white" h="72px" >
+            <Flex justify="space-between" w="100%" >
```

Comment:
Having a HStack wrap a flex is like wrapping a Flex in a Flex. We can probably get rid of this one

Reply:
Gone. Props moved to flex and still rendering.